data_safe_haven/config/sre_config.py

```diff
@@ -64,6 +64,7 @@ def template(cls: type[Self]) -> SREConfig:
                 data_provider_ip_addresses=[
                     "List of IP addresses belonging to data providers"
                 ],
+                gitea_servers="both/internal: whether to deploy both external and internal Gitea servers, or only an internal server",
```

Comment:
I suggest that the config option is called something more generic like `git_servers` in case we want to change which server we use in future.